package/utils/busybox/Makefile

```diff
@@ -139,8 +150,12 @@ define Package/busybox/install
 	$(INSTALL_DIR) $(1)/etc/init.d
 	$(INSTALL_DIR) $(1)/usr/sbin
 	$(CP) $(PKG_INSTALL_DIR)/* $(1)/
+ifdef CONFIG_BUSYBOX_CONFIG_FEATURE_SYSLOG
+	touch $(1)/etc/syslog.conf
```

Comment:
This change isn't really mentioned in the commit message.

Reply:
Fixed.

Reply:
Thanks, are SYSLOGD_CFG and SYSLOG redundant?

Reply:
Also this should be `FEATURE_SYSLOG` not just `SYSLOG`, please check `.config`

Reply:
Good catch. Done.

Reply:
> Thanks, are SYSLOGD_CFG and SYSLOG redundant?

I don't believe so.  You can have `syslogd` but it needs to be built with `syslog.conf` support, or else you can build `ksyslog` which also uses the file.